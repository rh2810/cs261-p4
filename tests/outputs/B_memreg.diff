--- outputs/B_memreg.txt	2017-11-09 17:26:44.688864247 -0500
+++ expected/B_memreg.txt	2017-11-03 11:51:30.000000000 -0400
@@ -0,0 +1,12 @@
+Beginning execution at 0x0100
+Y86 CPU state:
+  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
+  %rax: fedcba9876543210    %rcx: fedcba9876543210
+  %rdx: 0123456789abcdef    %rbx: 0000000000000200
+  %rsp: 0000000000000000    %rbp: 0000000000000000
+  %rsi: 0000000000000000    %rdi: 0000000000000000
+   %r8: 0000000000000000     %r9: 0000000000000000
+  %r10: 0000000000000000    %r11: 0000000000000000
+  %r12: 0000000000000000    %r13: 0000000000000000
+  %r14: 0000000000000000
+Total execution count: 6
