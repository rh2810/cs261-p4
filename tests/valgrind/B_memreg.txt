==18456== Memcheck, a memory error detector
==18456== Copyright (C) 2002-2015, and GNU GPL'd, by Julian Seward et al.
==18456== Using Valgrind-3.11.0 and LibVEX; rerun with -h for copyright info
==18456== Command: ../y86 -e inputs/memreg.o
==18456== 
==18456== 
==18456== HEAP SUMMARY:
==18456==     in use at exit: 0 bytes in 0 blocks
==18456==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==18456== 
==18456== All heap blocks were freed -- no leaks are possible
==18456== 
==18456== For counts of detected and suppressed errors, rerun with: -v
==18456== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
