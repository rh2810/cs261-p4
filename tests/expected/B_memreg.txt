Beginning execution at 0x0100
Y86 CPU state:
  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
  %rax: fedcba9876543210    %rcx: fedcba9876543210
  %rdx: 0123456789abcdef    %rbx: 0000000000000200
  %rsp: 0000000000000000    %rbp: 0000000000000000
  %rsi: 0000000000000000    %rdi: 0000000000000000
   %r8: 0000000000000000     %r9: 0000000000000000
  %r10: 0000000000000000    %r11: 0000000000000000
  %r12: 0000000000000000    %r13: 0000000000000000
  %r14: 0000000000000000
Total execution count: 6
